Mac/MainWindow/Timeline/TimelineViewController.swift

```diff
@@ -124,6 +124,7 @@ final class TimelineViewController: NSViewController, UndoableCommandRunner, Unr
 			}
 
 			directlyMarkedAsUnreadArticles = Set<Article>()
+			lastVerticlePosition = 0
```

Comment:
Nit — can be dealt with in future PR — `lastVerticlePosition` should be `lastVerticalPosition`